src/interfaces.js

```diff
@@ -0,0 +1,51 @@
+import {View} from './view';
+import {ViewStrategy} from './view-strategy';
+
+interface ComponentCreated {
+    /**
+    * Implement this hook if you want to perform custom logic after the constructor has been called.
+    * At this point in time, the view has also been created and both the view-model and the view 
+    * are connected to their controller. The hook will recieve the instance of the "owningView". 
+    * This is the view that the component is declared inside of. If the component itself has a view,
+    * this will be passed second.
+    */
```

Comment:
It might be worth documenting, that when a view is cached its created callback is only called once on the initial creation, not when it is reused.